components/portfolio/MarketPositions.tsx

```diff
@@ -125,197 +125,98 @@ export const MarketPositions = ({
     (pos) => pos.outcome == "Pool Share",
   );
 
-  return (
-    <div className={`${className}`}>
-      <MarketPositionHeader
-        marketId={market.marketId}
-        question={market.question}
-      />
-      <Table
-        columns={isLiquidityMarket ? COLUMNS_LIQUIDITY : COLUMNS}
-        data={positions.map<TableData>(
-          ({
-            assetId,
-            price,
-            userBalance,
-            outcome,
-            changePercentage,
-            market,
-            avgCost,
-            rpnl,
-            upnl,
-          }) => {
-            const baseAssetUsdPrice = lookUpAssetPrice(
-              market.baseAsset,
-              foreignAssetPrices,
-              usdZtgPrice,
-            );
+  //display balance if any of the positions is greater than 0.01 USD
+  const displayBalance = positions.some((pos) => {
```

Comment:
What happens if one of the balances in a market isn't above $0.01? Won't it still be displayed?

Reply:
Yes, good catch. I've made the appropriate changes in the latest push to hide rows and then entire tables if all the values are below the threshold.